packages/preferences/src/browser/preferences-decorator.ts

```diff
@@ -83,4 +83,23 @@ export class PreferencesDecorator implements TreeDecorator {
         }
         return defaultValue;
     }
+
+    private buildTooltip(data: PreferenceDataProperty): String {
+        let tooltips: string = '';
+        if (data.description) {
+            tooltips = data.description;
+        }
+        if (data.defaultValue) {
+            tooltips += `\nDefault: ${data.defaultValue}`;
```

Comment:
@lmcbout do you think we should add the following?
```ts
tooltips += `\nDefault: ${JSON.stringify(data.defaultValue)}`;
```

I tested it locally and it means that preferences with `default` values such as `""` and `[]` will be displayed, something the current implementation does not display properly. It also means that strings will be rendered as strings (ex: `"never"` instead of `never`)